### DISCUSSION TITLE
Universal Print - Please make this message user configurable

### DISCUSSION THREAD

**Discussion by podgaard** [microsoft/ALAppExtensions#14690] [Ideas]
When you print anything you get a confirmation message, could this please be disabled or made  into an option that can be disabled - like other messages can. 

Our first client that is trying Universal Print today just trashed this product, because that button gives the inventory guy 100 extra clicks each day, I also feel sorry for him. The message doesn't provide any value to user. 

"JobSentTxt: Label 'The print job %1 has been sent for processing in Universal Print.', Comment = '%1 = a print job ID.';"

https://github.com/microsoft/ALAppExtensions/blob/5939af7f78e22af893242f302cb5630eb9440fee/Apps/W1/MicrosoftUniversalPrint/src/UniversalPrintDocumentReady.Codeunit.al#L122

**Comment by JesperSchulz** [comment]
Any chance you could create a PR yourself to address your request? It would for sure be the quickest path to get this change into the product 😊

**Reply by podgaard** [reply]
I will have to learn to do a PR - excuse my ignorance, but it's not a process that I have worked much before with.

**Reply by JesperSchulz** [reply]
Sounds great! Here's some help to get you started: http://aka.ms/changealmodule.

**Reply by podgaard** [reply]
Sorry, I didn't have time to look into this yet and probably won't have time. My time belongs to hours I can invoice to clients. I think client is forced to pick another option for direct printing, also because that it can't recognize landscape prints from a BC report and the tray options don't appear to be fully implemented in the Universal Print or in the transfer from BC to UP.
